apps/microtvm/reference-vm/zephyr/base-box/base_box_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Usage: base_box_test.sh <TARGET>
+#     Execute microTVM Zephyr tests.
+#
+
+set -e
+set -x
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: base_box_test.sh <TARGET>"
+    exit -1
+fi
+
+target=$1
+
+pytest tests/micro/zephyr/test_zephyr.py --microtvm-platforms=${target}
+
+if [ $target == "nrf5340dk" ] || [ $target == "mps2_an521" ]; then
```

Comment:
i think this can be [ $target == "nrf5340dk" || $target == "mps2_an521" ], but would feel better about this being in python. or maybe like:

```
if [ $target == "stm32f746xx" ]; then
  echo "NOTE: skipped test_zephyr_aot.py on $target -- known failure"
else
  pytest ...
fi
```